tools/pepsi/src/main.rs

```diff
@@ -178,21 +189,21 @@ enum Command {
     Wireless(WirelessArguments),
 }
 
-async fn get_repository_root() -> anyhow::Result<PathBuf> {
-    let path = current_dir().context("Failed to get current directory")?;
+async fn get_repository_root() -> Result<PathBuf> {
+    let path = current_dir().wrap_err("failed to get current directory")?;
     let ancestors = path.as_path().ancestors();
     for ancestor in ancestors {
         let mut directory = read_dir(ancestor)
             .await
-            .with_context(|| format!("Failed to read directory {ancestor:?}"))?;
+            .wrap_err_with(|| format!("failed to read directory {ancestor:?}"))?;
         while let Some(child) = directory.next_entry().await.with_context(|| {
```

Comment:
I think you forgot a `wrap_err_with` here.

Reply:
Done